### ISSUE TITLE
About the Tensorflow C++ API documentation

### ISSUE THREAD

**Issue by yama-can** [tensorflow/tensorflow#60465]
The lack of documentation for the Tensorflow C++ API makes it difficult to build and use. Can you write how to use and build in README.md?

**Comment by gowthamkpr** [comment]
Hi @yama-can, thank you for creating an issue. can you please help us understands the specifics of the documentation and the tutorials that you want us to publish. Thanks!!

**Comment by yama-can** [comment]
of course.  Specifically, I would like you to describe the c++ api extension build method, include path and library path.

**Comment by anubhav1206** [comment]
Where exactly are we supposed to write it?

**Comment by Venkat6871** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The Tensorflow team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the Tensorflow space.
